client/src/components/navbar-component/DropdownItem.jsx

```diff
@@ -0,0 +1,33 @@
+import PropTypes from "prop-types";
+import "./Navbar.css";
+import ArrowForwardIcon from "@mui/icons-material/ArrowForward";
+import SettingsIcon from "@mui/icons-material/Settings";
+import LogoutIcon from "@mui/icons-material/Logout";
+import AdminIcon from "@mui/icons-material/Group";
+
+function DropdownItem({ label, keyword }) {
+  const icons = {
+    settings: <SettingsIcon color="action" />,
+    logout: <LogoutIcon color="action" />,
+    admin: <AdminIcon color="action" />,
+  };
+
```

Comment:
I would also recommend retrieving label from a dictionary that matches keyword to label as you have done with icons.